arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,618 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The SKU (pricing tier) of the account.",
+          "enum": [
+            "Free",
+            "Basic"
+          ],
+          "x-ms-enum": {
+            "name": "SkuNameEnum",
+            "modelAsString": true
+          }
+        },
+        "family": {
+          "type": "string",
+          "description": "sku family."
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "sku capcity."
+        }
+      },
+      "description": "The account sku."
+    },
+	"AutomationAccountUpdateProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "account sku."
+        }
+      },
+      "description": "The parameters supplied to the update account properties."
+    },
+    "AutomationAccountUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AutomationAccountUpdateProperties",
+          "x-ms-client-flatten": true,
+          "description": "account update properties."
+        },
+        "name": {
+          "type": "string",
+          "description": "name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "tags attached to the resource."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "The parameters supplied to the update automation account operation."
+    },
+    "AutomationAccountProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "sku of account."
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "last modified by"
+        },
+        "state": {
+          "type": "string",
+          "description": "status of account.",
+          "enum": [
+            "Ok",
+            "Unavailable",
+            "Suspended"
+          ],
+          "x-ms-enum": {
+            "name": "AutomationAccountState",
+            "modelAsString": true
+          }
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "creation time."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "last modified time."
+        },
+        "description": {
+          "type": "string",
+          "description": "description."
```

Comment:
`description` is not a sufficient description of the `description` field. I suggest a description that better articulates what the `description` field is describing.